### PR TITLE
I've added some comments to try and understand this block of code. 

### PR DESCRIPTION
**Pull request by HMassey**
My only confusion currently is where the word array is initialised. I understand it's utilisation fine, and I understand that it's coming from the noops API bit, I just can't see between the generateCompanyName function and the writeCompanyName function how it has the data.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
     });
 }
 
 function writeCompanyName(data)
 {
-    var companyNameH1 = $('#top-hook > h1');
+    var companyNameH1 = $('#top-hook > h1'); //make the heading a variable. This allows you to reinitialise
 
-    companyNameH1.text('');
+    companyNameH1.text(''); //returns the contents of the tag as a string? Not seen this method
```
Comment (dayallnash) [inline js/index.js]: actually it sets the element referenced by companyNameH1 to be an empty string - so it sets it to be from:

`<h1>We are the company that money</h1>`

to

`<h1></h1>`

File: js/index.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ function writeCompanyName(data)
                 if (x === 0) {
                     word = wordArr[x].toUpperCase();
                 } else {
-                    word += wordArr[x];
+                    word += wordArr[x]; //this whole block capitalises the first letter of the word right?
```
Comment (dayallnash) [inline js/index.js]: Yup

File: js/index.js [hunk 3]
```diff
@@ -1,20 +1,20 @@
 function generateCompanyName()
 {
     $.ajax({
-        url: 'api/words.php?companyName',
-        success: writeCompanyName
+        url: 'api/words.php?companyName', //url for API to ask for new words
+        success: writeCompanyName //execute the function if you get through
```
Comment (dayallnash) [inline js/index.js]: We could also have an 'error' param which executes a different function if there is an error returned by the API. We could write this error function to be generic enough to work for all of our AJAX request error handling. 'ajaxFailHandler' maybe?

Comment (HMassey) [reply js/index.js]: I'm guessing that's just following the syntax of 

url: 'api/words.php?companyName',
success: writeCompanyName,
error: ajaxFailHandler

But I don't know a meaningful way of representing the error! Perhaps changing all the tags to the same text?

Comment (dayallnash) [reply js/index.js]: Let's just console.log() the error text out for now.

**Comment by dayallnash** [comment]
Closing pull request as the comments aren't necessary and I think you understand it 👍

**Pull request closed without merging**